tests/vision/vision_conftest.py

```diff
@@ -45,10 +50,41 @@
            'coco_test_dataloader',
            'coco_test_visiondata',
            'two_tuples_dataloader',
-           'mnist_drifted_datasets'
+           'mnist_drifted_datasets',
+           'mock_trained_yolov5_object_detection',
+           'mock_trained_mnist',
            ]
 
 
+def _hash_image(image):
+    if isinstance(image, np.ndarray):
+        image = Image.fromarray(image)
+    elif isinstance(image, torch.Tensor):
+        image = Image.fromarray(image.cpu().numpy().squeeze())
+
+    image = image.resize((10, 10))
+    image = image.convert('L')
+
+    pixel_data = list(image.getdata())
+    avg_pixel = sum(pixel_data) / len(pixel_data)
+
+    bits = ''.join(['1' if (px >= avg_pixel) else '0' for px in pixel_data])
```

Comment:
why do you need this transformation?

Reply:
It's just a way to hash an image - if a pixel is above the avg - put 1, otherwise 0.

Reply:
but why a string of '0001011101' is better than '3023408509'?